hello-world-api/Startup.cs

```diff
@@ -38,7 +38,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.Use(async (context, next) =>

```

Comment:
Changing the secret key value to 'opensesame2' introduces a potential security risk. Please ensure this is reviewed thoroughly for any security implications.